src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
+  const today = date;
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
```

Comment:
I don’t fully understand why we need a link to an object here. I think that we can parse directly the object that was passed to us

```suggestion
  const dateISO = date.toISOString().slice(0, 10);
  const dateLocaleString = date.toLocaleDateString('en-US', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
  });
```